ws/src/main/scala/me/jeffmay/neo4j/client/rest/package.scala

```diff
@@ -1,8 +1,22 @@
 package me.jeffmay.neo4j.client
 
-import play.api.libs.json.{Json, Reads}
+import me.jeffmay.neo4j.client.cypher._
+import play.api.libs.json.{JsArray, Json, Reads, Writes}
```

Comment:
This is the `ws` project which uses Play Json and Play WS. I wrote a converter from `CypherValue` to `JsValue`